openlibrary/templates/type/edition/modal_links.html

```diff
@@ -9,22 +9,22 @@
 
   $ ga_attr = 'data-ol-link-track=%s' % ga_data if ga_data else ''
   <a href="$href" class="$link_class icon-link" $ga_attr>
-    <img class="icon-link__image" src="$img_src">
+    <img class="icon-link__image" src="$img_src" alt="$alt">
     <div class="icon-link__text">$link_text</div>
   </a>
 
 <div class="modal-links">
   $if work and work.get('key', ''):
-    $ review_content = icon_link("observations-modal-link", "/static/images/icons/reviews.svg", _("Review"), ga_data="ModalLinkClick|ReviewIcon")
+    $ review_content = icon_link("observations-modal-link", "/static/images/icons/reviews.svg", _("Review"), ga_data="ModalLinkClick|ReviewIcon", alt="")
     $:macros.ObservationsModal(work, review_content, 'sidebar-review')
 
     $:macros.StarRatings(work, edition)
 
-    $ notes_content = icon_link("notes-modal-link", "/static/images/icons/notes.svg", _("Notes"), ga_data="ModalLinkClick|NoteIcon")
+    $ notes_content = icon_link("notes-modal-link", "/static/images/icons/notes.svg", _("Notes"), ga_data="ModalLinkClick|NoteIcon", alt="")
     $:macros.NotesModal(work, edition, notes_content, 'sidebar-notes')
 
   $if ctx.get('share_links'):
-    $ share_content = icon_link("share-modal-link", "/static/images/icons/share.svg", _("Share"), login_redirect=False, ga_data="ModalLinkClick|ShareIcon")
+    $ share_content = icon_link("share-modal-link", "/static/images/icons/share.svg", _("Share"), login_redirect=False, ga_data="ModalLinkClick|ShareIcon", alt="")
     $ share_links = ctx.get('share_links')
     $:macros.ShareModal(share_links, page_url, share_content, "share-modal-wrapper")
-</div>
+</div>
```

Comment:
```suggestion
</div>

```
Don't remove the trailing newline character from files that you modify.